eng/scripts/RunHelix.ps1

```diff
@@ -6,7 +6,7 @@
 .PARAMETER Project
     The test project to publish and send to Helix.
 .PARAMETER HelixQueues
-    Set the Helix queues to use, the list is ';' separated.
+    Set the Helix queues to use. The list is '+' or ';'-separated.
```

Comment:
Supposedly Microsoft.DotNet.Helix.Sdk.MultiQueue.targets supports ','-separation but that doesn't seem to work